models/DataObject/Classificationstore/Dao.php

```diff
@@ -81,11 +85,11 @@ public function save()
                         $value = $fd->getDataForResource($value, null, []);
                         $this->model->setLocalizedKeyValue($groupId, $keyId, $value, $language);
                     } elseif ($fd instanceof DataObject\ClassDefinition\Data\EncryptedField) {
-                        $value = $fd->getDataForResource($value, $this->model->object, ['skipEncryption' => true]);
+                        $value = $fd->getDataForResource($value, $this->model->getObject(), ['skipEncryption' => true]);
```

Comment:
why not use newly created $object variable here?

Reply:
thx